### PR TITLE
Add Flake8 to travis CI

### LINKED ISSUE #14: Add flake8 to CI jobs

### PR DESCRIPTION
**Pull request by guilhermeleobas**
Fixes #14

### CODE REVIEW

**Comment by guilhermeleobas** [comment]
Currently, this only runs Travis on the `rbc/rbc` folder.

**Comment by pearu** [comment]
To fix travis CI failure, append  `# noqa: E<error id>` to the `setuptools` import line.
It could be good to keep also `setup.py` under flake8 validation.

**Comment by guilhermeleobas** [comment]
> To fix travis CI failure, append `# noqa: E<error id>` to the `setuptools` import line.
> It could be good to keep also `setup.py` under flake8 validation.

Oh, I was just testing to make sure `flake8` was working on CI.

**Comment by pearu** [comment]
Having
```
from setuptools import setup, find_packages  # noqa: E402
```
should make `flake8 .` working as well.

**Comment by guilhermeleobas** [comment]
`setup.py` is passing on flake8 in the other PR I have opened https://github.com/xnd-project/rbc/pull/12

**Comment by pearu** [comment]
I believe moving the setuptools import line in #12 should be undone as there was a reason for importing tools from setuptools later. I'll try to remember what was it..

**Comment by pearu** [comment]
The reason is documented here: https://github.com/scipy/scipy/blob/04c59ef48e66675f9902e88163506f3843fccdfb/setup.py#L88

File: .travis.yml [hunk 1]
```diff
@@ -23,9 +23,11 @@ install:
...
   - conda list
   - pip install thriftpy2
+  - pip install flake8
 
 before_script:
   - python setup.py develop
 
 script:
-  - pytest -v -r s rbc/
+  - flake8 rbc
```
Comment (pearu) [inline .travis.yml]: Change to `flake8 .`

**Pull request merged**